index.html

```diff
@@ -76,6 +76,10 @@
         <pre class="zenuml"></pre>
       </div>
     </div>
+    <!--  headlessui protal root hack: https://github.com/tailwindlabs/headlessui/discussions/666#discussioncomment-4966117  -->
```

Comment:
This adds a big burden to the client to use this library. For example, all the demo pages under `/cy` folder needs to be updated. Mermaid integration would also need to be updated. I am worried that if this design allows multiple ZenUML diagrams to be rendered on one page. Mermaid demo pages is such a case.

If I understand it correctly, all "clients" would need to have such an element for the choose theme to work. It breaks the API of:
1. const zenuml = new ZenUML(element)
2. zenuml.render(code, theme)
We need to find another way (maybe even hackier) to implement this. How about create this node at the begining?

Reply:
It still does not work on my computer. See the video below:

https://github.com/mermaid-js/zenuml-core/assets/169417/8ab63c67-a2fe-4871-b7bd-a87b8f6f8d72

Reply:
It seems that it has not been resolved.